data/strings/strings.txt

```diff
@@ -5254,6 +5254,18 @@
     zh-Hans = 语音语言
     zh-Hant = 語音語言
 
+  [pref_tts_test_voice_title]
+    comment = Settings «Route» category: «Tts Test Voice» title
+    tags = android,ios
```

Comment:
Since iOS doesn't have this problem, you can keep only ```tags = android```

Reply:
Corrected !!